src/backend/stream.rs

```diff
@@ -379,14 +379,14 @@ impl<'ctx> PulseStream<'ctx> {
                         s.set_state_callback(check_error, stm.as_mut() as *mut _ as *mut _);
                         s.set_write_callback(write_data, stm.as_mut() as *mut _ as *mut _);
 
-                        let buffer_size_bytes = latency_frames * stm.output_sample_spec.frame_size() as u32;
+                        let buffer_size_bytes = USecExt::to_bytes(latency_frames as u64, &stm.output_sample_spec) as u32;
```

Comment:
@Koalab99 have you checked if `buffer_size_bytes` value is different now?

Reply:
Yes it is,
Without the change, I got 8816 for the output and 4408 for the input, 
With the change, I got 384 for the output and 192 for the input.